README.md

```diff
@@ -47,17 +47,9 @@ See also environment variables from lower layers: [`docker-openjdk`](https://git
 
 | Variable                    | solrcore.property variable | java variable                                                | Default                                                      | Comments |
 | --------------------------- | --------------------------------- | ------------------------------------------------------------ | ------------------------------------------------------------ | --------------------------- |
-| SHARDING                    |                      |                                                              | false                                               | if true, configuration folders for shards will be created |
```

Comment:
This breaks backwards compatibility with previous images that were using sharding?

Reply:
Replying to more comments regarding backwards compatibility:

Indeed, backwards compatibility is broken. 
But, there is only one client using sharding atm. 
And nobody uses GLOBAL_ variables for solr in the same way as for Alfresco.